packages/pagination/src/usePagination.ts

```diff
@@ -5,17 +5,45 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import { useSelection } from '@zendeskgarden/container-selection';
+import {
+  useSelection,
+  IUseSelectionProps,
+  IUseSelectionState,
+  IGetItemPropsOptions
+} from '@zendeskgarden/container-selection';
 
-export function usePagination(options) {
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface IUsePaginationProps<Item> extends IUseSelectionProps<Item> {}
+
+export interface IGetPageProps<Item> extends IGetItemPropsOptions<Item> {
+  page?: number;
+  current?: any;
+  ariaLabel?: any;
+}
+
+export interface IGetContainerProps extends React.HTMLProps<any> {
+  role?: any;
```

Comment:
`role` is a typed attribute in `@types/react/index.d.ts`, however, our API needs to be able to pass a `null` value for this attribute.

Reply:
Could this be typed as `role | null` instead of `any`? Not sure if that's any clearer though...

Reply:
Yeah, it might be a little clearer, but still not great. Unfortunately, the API requires `null` to be a possible value for `role`. I typed it as `role | null`. Thanks for the suggestion 👍 .